### PR TITLE
Integer increment and number file view

### PR DESCRIPTION
**Pull request by courtfiala**
Increment works up to maxint, currently hard-coded at some big number

Getting number file redirects to view of text file containing number, can be easily changed to download file instead

### CODE REVIEW

**Comment by Batboa** [comment]
There is a pre-defined JavaScript called number so I think it is a bad idea to name one of our classes Number.

**Comment by courtfiala** [comment]
Whoops didn't realize that, renamed it to ImageNumber

File: examples/incrementImage/increment.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class Increment extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  /* iterate over canvas and set each pixel as random */
+  generatePixels(canvas, ctx, canvasData, incr) {
```
Comment (andykais) [inline examples/incrementImage/increment.jsx:10]: This code is repeated in generate.jsx

Comment (courtfiala) [reply examples/incrementImage/increment.jsx:10]: That was just me thinking of ideas for increment, didn't realize I pushed it. The whole directory can be deleted

**Comment by andykais** [comment]
I  just pushed to a branch [temp-modular-files](https://github.com/andykais/painting-base2/tree/temp-modular-files) that has a different structure. If you could incorporate just the `/client/libs/transformations.js` file, and move the array generation there, that would be much clearer. Then we can change that without affecting the `Generate` component which is much clearer

**Pull request merged**